docker-compose.yml

```diff
@@ -2,22 +2,19 @@ version: "3.6"
 
 services:
   web:
```

Comment:
make sure you update the server with this and see if you can deploy -- we have autodeployment, but you will still have to manually replace the `docker-compose.yml` on the server